### PR TITLE
feat(wip): forms

### PR DESCRIPTION
**Pull request by heydoyouknowme0**

### CODE REVIEW

**Review by GetPsyched — COMMENTED** [review]
This doesn't build?

**Comment by heydoyouknowme0** [comment]
Locally it does build

File: components/forms/schemas/form.ts
Comment (GetPsyched) [inline components/forms/schemas/form.ts:1]: This file isn't being used anywhere as it exists also as `schemas/forms.ts`

Moreover, do we really need this?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: components/forms/schemas/form.ts
Comment (heydoyouknowme0) [reply components/forms/schemas/form.ts:1]: its being used by both backend and frontend validation, I meant to cut it inside forms a it's better considering the scope duplicated accidentally across commits

**Comment by GetPsyched** [comment]
Split into:
- #197 
- #132
- #187
- #186

**Pull request closed without merging**